.github/workflows/test.yml

```
@@ -0,0 +1,35 @@
+name: Test
+
+on: push
+
+jobs:
+  test:
+    name: Bootstrap app, run linters and tests
```

Should we change to this to know the current Node version?
```suggestion
    name: Bootstrap app, run linters and tests with Node ${{ matrix.node }}
```

I know the current one also has the version, but should we specify it?